Backpack/src/main/res/values/backpack.text.xml

```diff
@@ -25,96 +25,115 @@
   <style name="bpkTextBase">
     <item name="android:fontFamily">sans-serif</item>
     <item name="android:textSize">16sp</item>
+    <item name="android:letterSpacing">-0.0125</item>
   </style>
 
   <style name="bpkTextCaps">
     <item name="android:fontFamily">sans-serif</item>
     <item name="android:textSize">10sp</item>
+    <item name="android:letterSpacing">0.04</item>
   </style>
 
   <style name="bpkTextLg">
     <item name="android:fontFamily">sans-serif</item>
     <item name="android:textSize">20sp</item>
+    <item name="android:letterSpacing">-0.02</item>
   </style>
 
   <style name="bpkTextSm">
     <item name="android:fontFamily">sans-serif</item>
     <item name="android:textSize">14sp</item>
+    <item name="android:letterSpacing">0</item>
   </style>
 
   <style name="bpkTextXl">
     <item name="android:fontFamily">sans-serif</item>
     <item name="android:textSize">24sp</item>
+    <item name="android:letterSpacing">-0.024999999999999998</item>
```

Comment:
Where are these numbers documented as the correct ones?

Reply:
those are type scale changes and are generated from tokens. I can confirm they are correct.